domains/osada.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "osadavc",
+    "email": "osadavidath@gmail.com",
+    "twitter": "osadavidath"
+  },
+  "record": {
+    "URL": "https://osada-is-a-dev.vercel.app"
```

Comment:
if you are doing an url record here, are you actually want an url redirect? please confirm!

Reply:
Yeah, A URL Redirect

Reply:
ok!